src/risk_atlas_nexus/library.py

```diff
@@ -13,6 +13,7 @@
 
 # workaround for txtai
 os.environ["KMP_DUPLICATE_LIB_OK"] = "True"
+os.environ["OMP_NUM_THREADS"] = "1"
```

Comment:
Is this for some multi-threaded operation? Can you point me where this is used?

Reply:
Hi @ingelise 
I was talking about this `os.environ["OMP_NUM_THREADS"] = "1`. It is working completely fine without this statement in my machine.

Reply:
Yes, pytest is successful.